FUNDERS.yaml

```diff
@@ -162,7 +162,7 @@ biont:
     funding_statement: |
       Co-funded by the European Union
 
-ELIXIR-UK-DaSH:
+elixir-uk:
```

Comment:
I know you've changed a lot of these, but, is there a reason for the change? Since this is a specific grant from elixir uk, it could make sense  to keep it as the more specific `elixir-uk-dash` and set `github: false` (which is what I'm assuming was the goal to fix the github link)

Reply:
Thank you, Helena, Yes. That was the goal. I will return it to how it was